test/Google.Api.Gax.Tests/ClientHelperTest.cs

```diff
@@ -127,7 +127,13 @@ private string RemoveUserAgent(Metadata metadata)
 
         private class DummySettings : ServiceSettingsBase
         {
-            public DummySettings Clone() => CloneInto(new DummySettings());
+            public DummySettings()
+            {
```

Comment:
Nit: Put open+close curly brackets on line above, as with the constructor below.